Lucidtech/Las/Client.cs

```diff
@@ -573,16 +865,89 @@ public object CreateBatch(string description)
             return ExecuteRequestResilient(RestSharpClient, request);
         }
 
+        /// <summary>Get information about a specific user, calls the GET /users/{user_id} endpoint.</summary>
+        /// <example>
+        /// <code>
+        /// Client client = new Client();
+        /// var response = client.GetUser("&lt;user_id&gt;");
+        /// </code>
+        /// </example>
+        /// <param name="userId">Id of the user</param>
+        /// <returns>User response from REST API</returns>
         public object GetUser(string userId) {
             var request = ClientRestRequest(Method.GET, $"/users/{userId}");
             return ExecuteRequestResilient(RestSharpClient, request);
         }
 
+        /// <summary>Delete the user with the provided user_id, calls the DELETE /users/{userId} endpoint.</summary>
+        /// <example>
+        /// <code>
+        /// Client client = new Client();
+        /// var response = client.DeleteUser("&lt;user_id&gt;");
+        /// </code>
+        /// </example>
+        /// <param name="userId">Id of the user</param>
+        /// <returns>User response from REST API</returns>
         public object DeleteUser(string userId) {
             var request = ClientRestRequest(Method.DELETE, $"/users/{userId}");
             return ExecuteRequestResilient(RestSharpClient, request);
         }
 
+        /// <summary>Updates a user, calls the PATCH /users/{userId} endpoint.</summary>
+        /// <example>
+        /// <code>
+        /// Client client = new Client();
+        /// var parameters = new Dictionary&lt;string, string&gt;{
+        ///     {"name", "User"}
+        /// };
+        /// var response = client.UpdateUser("&lt;user_id&gt;", parameters);
+        /// </code>
+        /// </example>
+        /// <param name="userId">Id of the user</param>
+        /// <param name="attributes">
+        /// Attributes to update.
+        /// Allowed attributes:
+        ///     name (string),
+        ///     avatar (base64-encoded image)
```

Comment:
Are we a bit inconsisten in how we name optional params to a function?
attributes
optionalParams
etc